ros2_kdl_node:
  ros__parameters:
    # Trajectory parameters
    traj_duration: 1.5
    acc_duration: 0.5
    total_time: 1.5
    trajectory_len: 150
    Kp: 1.0

    # End-effector target position
    end_position_x: 0.3
    end_position_y: 0.0
    end_position_z: 0.4

    # Command interface and control
    cmd_interface: "velocity"        # Options: "position", "velocity", "effort"
    ctrl: "velocity_ctrl"            # Options: "velocity_ctrl", "velocity_ctrl_null", "vision_ctrl"

    # Trajectory type and s_type
    traj_type: "linear"              # Options: "linear", "circular"
    s_type: "trapezoidal"            # Options: "trapezoidal", "cubic"

